### DISCUSSION TITLE
google map with map chart on it.

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9375] [CLI - PWA mode] [answered]
Hello,
![image](https://user-images.githubusercontent.com/57760543/119224322-7b2d2780-bb1b-11eb-88b6-b6ce109f5c13.png)

I was trying to use the Google Maps API and was using some extensions to get the google map but I wanted to have the map chart over it according to the different zip codes of that state.
So can anyone pls help how can these be done

**Comment by ontwikkelfabriek** [comment, marked as answer]
@bagladivyang03 

There are many ways to archive an overlay( = the semi transparent blue polygons you see, in this case called a choropleth )  on a Google maps base layer other than the Google Maps Api. 

But since you asked for using a Google api based solution I would take a look at this Vue Google maps library:
https://github.com/diegoazh/gmap-vue

Especially take a look at the examples to see if you can create an overlay with filled polygons like the zip code example  of your image.( you'll need an api key from google, very easy to obtain )

Examples:
https://diegoazh.github.io/gmap-vue/examples/

If you want a more comprehensive way of displaying GIS data on web pages you should read up on Geoserver, Openlayers, Leaflet, Mapbox, ect.

**Reply by bagladivyang03** [reply]
Thanks, @ontwikkelfabriek for the resources I have done that using different markers according to zip codes.
